e2e-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cypress/base:20.18.0
+FROM cypress/base:22.11.0
```

Comment:
IDK if we should upgrade to Node 22 just yet.
Cypress publishes 20.x.x and 22.x.x images regularly, first number means Node runtime.

Reply:
yeah, probably right, we can wait a bit for new minor version